engine/redis-helper.js

```diff
@@ -0,0 +1,24 @@
+import redis from 'redis';
+
+function getClient(dbTestNumber) {
+  const client = redis.createClient(process.env.REDISCLOUD_URL, { no_ready_check: true });
+  return new Promise((resolve, reject) => {
+    if (dbTestNumber) {
+      client.select(dbTestNumber, (err) => {
+        if (err) {
+          reject(err);
+        }
+        resolve(client);
+      });
+    } else {
```

Comment:
nit: To me, this would look cleaner as:

    if (!dbTestNumber) {
      return resolve(client);
    }

    client.select(dbTestNumber, (err) => {
      return err ? reject(err) : resolve(client);
    }

Feel free to implement some or none of this suggestion!